### DISCUSSION TITLE
OpenELB Not Functioning Outside Kubernetes After Upgrade to Kubernetes v1.25.16

### DISCUSSION THREAD

**Discussion by rkazak07** [openelb/openelb#385] [General]
Hello,

After upgrading our Kubernetes cluster from version 1.24.1 to 1.25.16, we have encountered an issue where OpenELB no longer functions outside the Kubernetes cluster. Inside the cluster, services seem to be running as expected, but external access through OpenELB is not working.

### Environment
- OpenELB version: v0.5.0
- OS: Debian11

**Comment by renyunkang** [comment]
Which mode provided by openelb are you using? 
Is there any error log information reported by openelb-manager?

**Reply by rkazak07** [reply]
I'm using the layer 2 method. In OpenElb Manager, I am repeatedly receiving the following logs.
I'm using cilium as the CNI
version: v1.13.0
---

{"level":"info","ts":1709551173.830806,"logger":"arpSpeaker","msg":"got ARP request, sending response","interface":"ens192","ip":"10.100.36.101","senderIP":"10.100.36.1","senderMAC":"00:29:1f:09:03:4c","responseMAC":"00:51:59:a2:54:9a"}
{"level":"info","ts":1709551201.9906914,"logger":"arpSpeaker","msg":"got ARP request, sending response","interface":"ens192","ip":"10.100.36.109","senderIP":"10.100.36.1","senderMAC":"00:29:1f:09:03:4c","responseMAC":"00:54:52:9:2d:f3"}
{"level":"info","ts":1709551203.2607512,"logger":"arpSpeaker","msg":"got ARP request, sending response","interface":"ens192","ip":"10.100.36.100","senderIP":"10.100.36.1","senderMAC":"00:29:1f:09:03:4c","responseMAC":"00:53:52:a6:a7:fs"}

--
thanks

**Reply by renyunkang** [reply]
According to the log, it seems that there is no problem with openelb-manager. This is a normal reply to the arp request.

Have there been changes to the cluster nodes?

**Reply by rkazak07** [reply]
Hello,
thank you for you support. I resolved the issue by modifying the "KubeProxyReplacement: strict" parametrer within the configmap of the cilium CNI.
